packages/quick_actions/lib/quick_actions.dart

```diff
@@ -54,9 +65,14 @@ class QuickActions {
       assert(call.method == 'launch');
       handler(call.arguments);
     });
+    // ignore: deprecated_member_use_from_same_package
     runLaunchAction(handler);
   }
 
+  /// This is only exposed for the unit tests. Do not rely on it, it may break
+  /// without warning in the future.
+  @visibleForTesting
+  @deprecated
```

Comment:
Since we're making a breaking change anyway (removing `channel`) and this is a pre-1.0 plugin I think that we can probably skip the deprecated step and just remove this from the public API.